logging/config.go

```diff
@@ -249,3 +251,9 @@ func ConfigToJSON(cfg *Config) (string, error) {
 	})
 	return string(jsonCfg), err
 }
+
+func stackdriverConfig() zap.Config {
```

Comment:
Why is this called `stackdriverConfig`?

Reply:
`zapdriver` is specific to stack driver - I figured that'd be more explicit than `zapdriverConfig`